distro.py

```diff
@@ -1074,7 +1074,71 @@ def _parse_distro_release_content(line):
         return distro_info
 
 
-_distro = LinuxDistribution()
+def _normalize_id(distro_id, table):
+    """ Helper function which normalizes a distro id value. """
+    distro_id = distro_id.lower().replace(' ', '_')
+    return table.get(distro_id, distro_id)
+
+
+class LinuxMintDistribution(LinuxDistribution):
+    """ LinuxMint will have the same /etc/os-release file
+    as it's upstream Ubuntu so we should not use any info
+    from that file except `ID_LIKE`. """
+    def os_release_attr(self, attribute):
+        if attribute == 'id_like':
+            v = super(LinuxMintDistribution, self).os_release_attr(attribute)
```

Comment:
This would create a linting error because there is a hanging indent that has a multiple of 4 spaces that's not a code indent. 

This type of thing isn't optimized away in byte-code, two extra codes for `STORE FAST` and `LOAD FAST` before the return.